### PR TITLE
UG: Added a few more examples for other commands and change formatting

### PR DESCRIPTION
**Pull request by rsanchez-macias**
Let me know if I should reword anything, I will reread and make changes after working on DG

### CODE REVIEW

**Comment by rsanchez-macias** [comment]
A preview of the changes:

Screen Shot 2020-04-10 at 10 10 38 PM

**Comment by rsanchez-macias** [comment]
All commands follow same structure

File: docs/UserGuide.md [hunk 1]
```diff
@@ -18,234 +19,280 @@ By: `AY1920S2-CS2113-T14-3`
...
-            * `add n/Project Meeting t/12:00-13:00 15:00-16:00 d/2020-07-01 2020-09-01 l/NUS NTU c/meeting`
-            * `add n/2113 v2.1 t/23:00-24:00 d/2020-05-16 c/deadline`
-            * `add n/Project Meeting t/12:00-13:00 15:00-16:00 d/2020-10-01 2020-10-04 l/NUS NTU`
-     * When adding class:
-        * `<DATE>` should be which day in a week, represented by integer (e.g. `1 3` means Mon Wed). 
-        * Examples: 
-            * `add t/11:00-12:00 01:00-03:00 n/2113 d/3 4 c/CLASS l/COM2 COM1`
-            * `add n/3245 t/17:00-19:00 d/5 c/CLASS`
+Users add tasks using this command. <br/>
+**Usage**: `add  n/<NAME> t/<TIME> l/<LOCATION> d/<DATE> i/<INFORMATION> r/<REMINDER> c/<CATEGORY>` <br/>
```
Comment (JTWang2000) [inline docs/UserGuide.md:70]: This looks much better than before! Thanks!

**Review by JTWang2000 — COMMENTED** [review]
Thanks for your effort! This looks much better than before!

**Pull request merged**